### PR TITLE
Unify names of time precision constants

### PR DESCRIPTION
**Pull request by thiemowmde**
Same patch in PHP: https://gerrit.wikimedia.org/r/#/c/194306/

[Bug: T99911](https://phabricator.wikimedia.org/T99911)

### CODE REVIEW

File: src/values/TimeValue.js [hunk 1]
```diff
@@ -294,13 +294,13 @@ SELF.getCalendarModelTextByUri = function( uri ) {
  * @since 0.7
  */
 SELF.PRECISIONS = [
-	{ id: 'GY', text: 'billion years' },
-	{ id: 'MY100', text: 'hundred million years' },
-	{ id: 'MY10', text: 'ten million years' },
-	{ id: 'MY', text: 'million years' },
-	{ id: 'KY100', text: '100,000 years' },
-	{ id: 'KY10', text: '10,000 years' },
-	{ id: 'KY', text: 'millenium' },
```
Comment (JeroenDeDauw) [inline src/values/TimeValue.js]: Are these not used anywhere? Also, is this change consistent with what we have in PHP?

And finally, is this really better? Sure, it's the same format, though is it how those things are normally written? I suspect not, which is supported by http://en.wiktionary.org/wiki/Gyr

Comment (thiemowmde) [inline src/values/TimeValue.js]: Currently these JavaScript constants are not used anywhere. They were just introduced in the unreleased 0.7.0.

PHP constants are named different, see https://gerrit.wikimedia.org/r/#/c/194306/. This is the main reason why I touched this: 1. PHP constants use lower (!) case characters. 2. PHP and JavaScript constants are named differently.

If there is a better naming scheme, I'm open for suggestions.

**Comment by JeroenDeDauw** [comment]
Don't forget to update the release notes. And perhaps the @since tag for the SELF.PRECISIONS constant should be updated as well

**Comment by thiemowmde** [comment]
Done in #83, thanks.

**Pull request merged**